pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
I agree: something like thats what I had in mind, except i wanted to use the antrun plugin so that you'd actually see the output from elasticsearch. Somehow this isn't bothering anyone all the output is "missing" (delayed 15 minutes or whatever crazy shit happens with mvn exec buffering), but it makes things undebuggable.

With antrun, we could also do stuff like unzip the es jar from target/releases/... (which is manually what i do, if i want to manually run ES from a source checkout) and run bin/elasticsearch....